back/app/services/side_fx_initiative_service.rb

```diff
@@ -109,7 +109,7 @@ def before_publish(initiative, _user)
   end
 
   def after_publish(initiative, user)
-    add_autoreaction initiative
+    add_autoreaction initiative, user
```

Comment:
```suggestion
    add_autoreaction(initiative, user) if user.id == initiative.author_id
```

Reply:
Nice!

Reply:
Except... I think we do want to add the vote if the anonymous user is permitted to vote 🤔